### PR TITLE
samples: Add console suspend in system_off 

### PR DESCRIPTION
**Pull request by maje-emb**
Added console suspend before `sys_poweroff` to reduce power usage in 
`bluetooth: peripheral_power_profiling` and `nfc: system_off` samples.

### CODE REVIEW

**Review by dchat-nordic — APPROVED** [review]

File: samples/bluetooth/peripheral_power_profiling/prj.conf [hunk 1]
```diff
@@ -31,5 +31,10 @@ CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_FLASH_MAP=y
 
 CONFIG_POWEROFF=y
+CONFIG_PM_DEVICE=y
 
 CONFIG_MAIN_STACK_SIZE=2048
+
+CONFIG_ASSERT=n
+CONFIG_BT_ASSERT=n
+CONFIG_ASSERT_VERBOSE=n
```
Comment (grochu) [inline samples/bluetooth/peripheral_power_profiling/prj.conf:40]: Should we consider disabling `CONFIG_NCS_SAMPLES_DEFAULTS`?

**Review by grochu — COMMENTED** [review]
Leaving a configuration-related suggestion.

File: samples/bluetooth/peripheral_power_profiling/prj.conf (hunk 1 above)
Comment (maje-emb) [reply samples/bluetooth/peripheral_power_profiling/prj.conf]: > Should we consider disabling `CONFIG_NCS_SAMPLES_DEFAULTS`?

Yes, I think we should disable `CONFIG_NCS_SAMPLES_DEFAULTS`.

Comment (maje-emb) [reply samples/bluetooth/peripheral_power_profiling/prj.conf]: > Why you are disabling asserts?

I disabled asserts to reduce code and power consumption.

**Review by doki-nordic — APPROVED** [review]

**Pull request merged**